src/geo_types_to_wkt.rs

```diff
@@ -42,11 +43,12 @@ where
 ///
 /// let point: Point<f64> = point!(x: 1., y: 2.);
 ///
-/// assert_eq!(point.wkt_string(), "POINT(1 2)");
+/// assert_eq!(point.wkt_string(), "POINT(1.0 2.0)");
```

Comment:
I don't know how to feel about this change in behavior. On the one hand it makes our serialization less efficient which is bad, OTOH it conveys to the reader that this was serialized from floating point, so I guess that's useful.

Reply:
In the interest of not leaving ambiguous comments, I'll say I prefer the old version, which serialized efficiently, but I'd be willing to let that small demerit go if there are substantial other wins.

That said, if there's a way to use faster float serialization while still maintaining the integer display for .0, I think that'd be best.

Reply:
`ryu` doesn't have _any_ options as far as I can tell. So if we switched to ryu I don't think we'd be able to serialized floats without the `.0`

Reply:
We can still check if the string ends with `.0` and strip that off.